### DISCUSSION TITLE
is it possible to hide or block ABORT button once the playbook is already running in AWX??

### DISCUSSION THREAD

**Discussion by dariopai** [ansibleguy76/ansibleforms#226] [Q&A]
is it possible to hide or block ABORT button once the playbook is already running in AWX?? since I dont want users to stop a playbook in the middle of an execution due to changes in target devices go wrong.

**Comment by ansibleguy76** [comment]
can you make that an issue.  would be rather simple

**Comment by ansibleguy76** [comment]
transfered to issues
